front/app/components/admin/Graphs/BarChartByCategory.tsx

```diff
@@ -4,7 +4,7 @@ import { isEmpty } from 'lodash-es';
 
 // intl
 import { useIntl } from 'utils/cl-intl';
-import messages from '../../messages';
+import messages from '../../../containers/Admin/dashboard/messages';
```

Comment:
I moved this component into the main components folder, since now it's used in two locations. If I move the messages though they will have to be re-translated :thinking: Are we ok leaving them like this? Or should I make a new messages file and re-translate them?

Reply:
I think since in both cases they're in a subfolder of an admin folder, they will not be retranslated if you move them (as long as you don't change their IDs). You could confirm that by moving them and pushing, and then see if crowdin deletes and adds them again

Reply:
Unfortunately it looks like the messages are used in multiple files, so I can't move them without needing to create a separate key :thinking: I'll leave them where they are for then I suppose.